File: sem3/map/ToyLanguageInterpreter/src/view/commands/ExitCommand.java
package view.commands;

import utils.exceptions.ExitTextMenuException;
import utils.exceptions.InterpreterException;

import java.util.Scanner;

/**
 * Created by mirko on 14/11/2016.
 */
public class ExitCommand extends Command {
    public ExitCommand(String key, String description) {
        super(key, description);
    }

    @Override
    public void execute(Scanner scanner) throws InterpreterException {
        throw new ExitTextMenuException("");
    }
}

File: sem3/map/ToySharpLanguageInterpreter/ToySharpLanguageInterpreter/src/view/commands/ExitCommand.cs
using System;
using System.IO;

namespace ToySharpLanguageInterpreter
{
	public class ExitCommand : Command
	{
		public ExitCommand(string key, string description) : base(key, description)
		{
		}


		public override void execute()
		{
			throw (new ExitTextMenuException(""));
		}
	}
}